README.md

```diff
@@ -1,2 +1,43 @@
-# coretipjar
-core tipjar details
+# Dogecoin Core Earmarked Developer Funds
+
+This repository holds details of the multisig address for Dogecoin Core Earmarked Developer Funds.
+
+## DRAFT Proposal
+
+TODO: Remove draft label once everyone has confirmed this text.
+
+This proposal is for a new segregated multi-signature wallet, designed to
+transparently manage a sum of 5,000,000 DOGE. The DOGE held in this wallet
+will disburse rewards to developers of Dogecoin Core for work on all
+contributions, no matter how big or small.
+
+We propose to create a multi-signature wallet, operated by Dogecoin Core
+developers and members of the Dogecoin Foundation. The wallet shall be secured
+with a three out of five signature threshold, with the following custodians:
+
+* Michi Lumin (Dogecoin Core developer, Dogecoin Foundation Board)
+* Ross Nicoll (Former Dogecoin Core developer)
+* Patrick Lodder (Dogecoin Core developer)
+* Marshall Hayner (Dogecoin Foundation Board)
+* Chromatic (Dogecoin Core developer)
```

Comment:
I'm little-c chromatic; you might be thinking of my father, Big-C Chromatic :zany_face:

Reply:
:D Fixed